viewer/src/datamodels/cad/rendering/EffectRenderManager.ts

```diff
@@ -0,0 +1,132 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
+import { MaterialManager } from '../MaterialManager';
+import { RenderMode } from './RenderMode';
+import * as THREE from 'three';
+import { Vector2, DataTexture } from 'three';
+import { edgeDetectionShaders } from './shaders';
+import { CogniteColors } from '@/utilities';
+
+export class EffectRenderManager {

```

Comment:
Suggest we rename this to RenderManager and remove `RenderManager` interface. `RevealManager` can have a `render()`-function which triggers rendering using this class.

Reply:
Skipped this as there are many usages of RenderManager from examples, so I feel that the refactoring is not fitting to have in this PR.